src/commands/new/selector.ts

```diff
@@ -22,7 +23,10 @@ module.exports = {
     try {
       const options = args.parseOptions(SELECTOR_OPTIONS);
       const outputDirectory = args.getOutputDirectory();
-      const filters = JSON.parse(parameters.options.filters) || [];
+      const filters = typeof parameters.options.filters === "string" ?
```

Comment:
I posted this in Alex's PR, but I think the easiest way is
```suggestion
      const filters = JSON.parse(parameters.options.filters || '[]')
```

Reply:
The issue is that when we load the file using toolbox.filesystem.read(options.fileName, 'json'), it formats the entire body of the JSON file, and when you try to parse valid json it throws an error